src/screenshot.js

```diff
@@ -22,6 +23,7 @@ module.exports = {
     }
     await page.setContent(html, { waitUntil })
     const element = await page.$('body')
+    await beforeScreenshot(page)
```

Comment:
Could you add a test in `screenshot.spec.js` that ensures this feature works?

Reply:
Will do 👍